packages/config/config/devices/0x0084/mimolite.json

```diff
@@ -64,7 +65,7 @@
 			"label": "Clear_Pulse_Meter_Counts",
 			"valueSize": 1,
 			"minValue": 0,
-			"maxValue": 255,
+			"maxValue": 0,
```

Comment:
Can you really only set 0 here? Then I'd probably also set `allowManualEntry: false`

Reply:
Yes, it seems to clear the counter when you send zero (as far as I can tell)